tests/exporters/test_onnx_export.py

```diff
@@ -139,6 +139,7 @@ def _get_models_to_test(export_models_dict: Dict):
     models_to_test = []
     if is_torch_available() or is_tf_available():
         for model_type, model_names_tasks in export_models_dict.items():
+            model_type = model_type.replace("_", "-")
```

Comment:
Why?

Reply:
I don't really understood what was happening, but on `transformers` side this is done: https://github.com/huggingface/transformers/blob/d87e381f9303c7d6a8aa7333dc09ed767de7395f/src/transformers/onnx/features.py#L723 
cc @fxmarty

Reply:
yes there is an unneeded `model_type = model_type.replace("_", "-")` in `get_exporter_config_constructor`